### PR TITLE
[15.0][MIG] account move sent usability

### PR DESCRIPTION
**Pull request by IT-Ideas**

### CODE REVIEW

**Comment by IT-Ideas** [comment]
@alexis-via

**Comment by IT-Ideas** [comment]
Hey @alexis-via who else could have a look to the PR?
Cc @rousseldenis

File: account_move_sent_usability/views/account_move.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <field name="name">account.move.filter</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_invoice_filter" />
+        <field name="arch" type="xml">
+
+            <xpath expr="//filter[@name='open']/preceding::separator" position="before">
+                <separator />
+                <filter
+                    name="unsent"
+                    string="Unsent"
```
Comment (sbidoul) [inline account_move_sent_usability/views/account_move.xml]:
```suggestion
                    string="Not Sent"
```

**Review by sbidoul — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Comment by IT-Ideas** [comment]
@rafaelbn i'll take care of the rebase and will take @sbidoul suggestion into account.

**Comment by IT-Ideas** [comment]
@rafaelbn rebase has been done 🙂

**Pull request merged**